buildtap.py

```diff
@@ -46,8 +49,8 @@ def __init__(self, opt):
         self.cert_pw = opt.certpw
         self.crosscert = os.path.join(self.top, opt.crosscert)
 
-        self.inf2cat_cmd = os.path.join(self.ddk_path, 'bin', 'selfsign', 'Inf2Cat')
-        self.signtool_cmd = os.path.join(self.ddk_path, 'bin', 'x86', 'SignTool')
+        self.inf2cat_cmd = 'Inf2Cat.exe'
+        self.signtool_cmd = 'SignTool.exe'
```

Comment:
The SetupBuildEnv.cmd script ensures that these are in PATH, right?

Reply:
Yes. Otherwise I have to do path parsing to get version numbers and...that would be unfortunate.

This is, IIRC, different from the DDK where path computation made more sense.